dockerfiles/neuroconv_developer_build_auto_config

```diff
@@ -0,0 +1,8 @@
+FROM python:3.10
```

Comment:
@CodyCBakerPhD 
usually changing to a -slim tag (e.g. `python:3.10.12-slim`) can substantially cut the final image size
slim images do come with some limitations in libraries, but these tend to be unnecessary for production images, I think it's worth a try

Reply:
Swapped all to slim